addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -20,30 +20,58 @@ sAnimations.registry.ProductWishlist = sAnimations.Class.extend(ProductConfigura
     },
     events: sAnimations.Class.events,
 
-    start: function () {
+    /**
+     * Init variables.
+     *
+     * @override
+     */
+    init: function (parent) {
+        this._super.apply(this, arguments);
+        this.wishlistProductIDs = [];
+    },
+    /**
+     * Get the current wishlist items.
+     *
+     * @override
+     * @returns {Deferred}
```

Comment:
`@override` is enough to know it returns a deferred, so remove the `@returns`.

Reply:
ok